apps/dataset/serializers/file_serializers.py

```diff
@@ -77,9 +77,11 @@ def get(self, with_valid=True):
             file = QuerySet(File).filter(id=file_id).first()
             if file is None:
                 raise NotFound404(404, "不存在的文件")
-            # 如果是mp3文件，直接返回文件流
-            if file.file_name.split(".")[-1] == 'mp3':
-                return HttpResponse(file.get_byte(), status=200, headers={'Content-Type': 'audio/mp3',
-                                                                          'Content-Disposition': 'attachment; filename="abc.mp3"'})
+            # 如果是音频文件，直接返回文件流
+            file_type = file.file_name.split(".")[-1]
+            if file_type in ['mp3', 'wav', 'ogg', 'aac']:
+                return HttpResponse(file.get_byte(), status=200, headers={'Content-Type': f'audio/{file_type}',
+                                                                          'Content-Disposition': 'attachment; filename="{}"'.format(
+                                                                              file.file_name)})
             return HttpResponse(file.get_byte(), status=200,
                                 headers={'Content-Type': mime_types.get(file.file_name.split(".")[-1], 'text/plain')})
```

Comment:
The provided code looks generally structured correctly but could be optimized and improved for better readability and maintainability. Here are some suggestions:

1. **String Formatting**: The `Content-Disposition` header uses an older Python string formatting method (`str.format()`). In newer versions of Python (Python 3.6 and later), you can use f-strings for more modern syntax:

   ```python
   ... = HttpResponse(
       file.get_byte(),
       status=200,
       headers={
           'Content-Type': f'audio/{file_type}',
           'Content-Disposition': f'attachment; filename="{file.file_name}"'
       }
   )
   ```

2. **List Check in IF Statement**: You currently check if the file type is one of specific values directly. This makes sense, but consider using a pre-defined list of audio types to make it easier to extend or modify without changing the logic.

   ```python
   AUDIO_TYPES = ['mp3', 'wav', 'ogg', 'aac']
   if file_type in AUDIO_TYPES:
       ...
   ```

3. **Consistent Variable Naming**: Ensure that variable names are consistently named throughout the function. For example, `file_id` seems like it should be `file_id_str` since `id` might not always be numeric.

By implementing these changes, the code becomes slightly cleaner and more readable while maintaining its functionality.